Tests/Fixtures/app/config/packages/rollbar.yml

```diff
@@ -0,0 +1,5 @@
+rollbar:
+    enabled: true
+    access_token: ~ # pulled from phpunit.xml
+    environment: 'rollbar-php-symfony-test-app'
+    minimum_level: 'info'
```

Comment:
please add an empty line.

Reply:
Is this the reason this PR was never merged?